build_scripts/windows/scripts/az.completion.psm1

```diff
@@ -0,0 +1,18 @@
+Register-ArgumentCompleter -Native -CommandName az -ScriptBlock {
+    param($commandName, $wordToComplete, $cursorPosition)
```

Comment:
Dotnet provide an example of how to write Register-ArgumentCompleter : https://learn.microsoft.com/en-us/dotnet/core/tools/enable-tab-autocomplete?WT.mc_id=modinfra-35653-salean#powershell